extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/controlBar.js

```diff
@@ -11,6 +11,7 @@ var React = require('react'),
     Popover = require('../views/popover'),
     VideoQualityPanel = require('./videoQualityPanel'),
     ClosedCaptionPopover = require('./closed-caption/closedCaptionPopover'),
+    ConfigPanel = require('./configPanel'),
     Logo = require('./logo');
     Icon = require('./icon');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "let", "const" or "var" keyword to this declaration of "Icon" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)